pages/api/Schema/CoachingAnswerSets/dataHandler.ts

```diff
@@ -74,22 +72,17 @@ const getCoachingAnswerSets = (
     const answers = createCoachingAnswersList(answerSetData, included);
     //create questions
     const questions = createCoachingQuestionsList(answerSetData, included);
-
-    const answerSet: CoachingAnswerSet = {
+    debugger;
```

Comment:
```debugger``` here. Could probably squash the commit after removing it.

Reply:
👍 PR looks good to me. I'm happy to approve it after this is removed :) Or anyone else can approve it.